src/rebuild/srv.c

```diff
@@ -1069,6 +1069,53 @@ rebuild_task_destroy(struct rebuild_task *task)
 	D_FREE(task);
 }
 
+/* Try merge the tasks to the current task.
+ *
+ * return 1 means merging the rebuild target to other task.
+ * return 0 means these targets can not merge.
+ *
+ **/
+static int
+rebuild_try_merge_tgts(const uuid_t pool_uuid, uint32_t map_ver,
+		       struct pool_target_id_list *tgts_failed)
+{
+	struct rebuild_task *task;
+	struct rebuild_task *found = NULL;
+	int		    rc;
+
+	d_list_for_each_entry(task, &rebuild_gst.rg_queue_list,
+			      dst_list) {
+		if (uuid_compare(task->dst_pool_uuid, pool_uuid) == 0) {
+			found = task;
+			break;
+		}
+	}
+
+	if (found == NULL)
+		return 0;
```

Comment:
Why can't the rebuild being retried when there isn't pending rebuild task for the same pool? I don't quite understand this.